src/python_bindings/py_data_stats.h

```diff
@@ -6,13 +6,48 @@
 
 namespace python_bindings {
 
+namespace py = pybind11;
+
 class PyDataStats : public PyAlgorithm<algos::DataStats, PyAlgorithmBase> {
     using PyAlgorithmBase::algorithm_;
 
 public:
     [[nodiscard]] std::string GetResultString() const {
```

Comment:
I'd rename this to something like `GetAllStatisticsAsString`, `ResultString` sounds to vague